test/functional/apps/dashboard/_dashboard_time.js

```diff
@@ -42,8 +42,8 @@ export default function ({ getPageObjects, getService }) {
       it('is saved', async () => {
         await PageObjects.dashboard.clickNewDashboard();
         await PageObjects.dashboard.addVisualizations([PageObjects.dashboard.getTestVisualizationNames()[0]]);
-        const isDashboardSaved = await PageObjects.dashboard.saveDashboard(dashboardName, { storeTimeWithDashboard: false });
-        expect(isDashboardSaved).to.eql(true);
+        // saveDashboard asserts that it saves successfully.
+        await PageObjects.dashboard.saveDashboard(dashboardName, { storeTimeWithDashboard: false });
```

Comment:
Maybe rename the saveDashboard method to make it clear that it ensures the save, since the same comment is used in more than one place to indicate that?